README.md

```diff
@@ -122,6 +123,7 @@ If you like this project, please consider giving it a star.
 
 ## 📝 License <a name="license"></a>
 
+
 This project is [MIT](https://github.com/faisalakb/Ruby_Morse_Code.git/blob/dev/LICENSE) licensed.
```

Comment:
Great job team 🤝

- [ ] Kindly fix the broken MIT license link to make your readme file more professional.
![MiT license](https://github.com/faisalakb/Ruby_Morse_Code/assets/63917725/523ae31e-5b31-41b1-b5ba-03603165dacf)